### DISCUSSION TITLE
PrimeReact 10.0.1 and Vite

### DISCUSSION THREAD

**Discussion by zorbill** [primefaces/community#144] [PrimeReact] [answered]
I was very excited to see the new Tailwind PrimeReact today and decided to give it a try, but ran into a few issues.   I think I jury-rigged my way by them, but wanted you to know.

I'm trying PrimeReact 10.0.1 with vite 

First i got a a bunch like this...
[Failed to resolve entry for package "D:\Projects\React\whit\node_modules\primereact\api". The package may have incorrect main/module/exports specified in its package.json. [plugin vite:dep-scan]]

I was able to get by these errors by changing esm.js to esm.mjs in the package.json in node_modules/primereact/api  (there was no api.esm.js file).  I did this for all the package.json files in the primereact folder.

```
{
    "main": "./api.cjs.js",
    "module": "./api.esm.mjs",
    "unpkg": "./api.min.js",
    "types": "./api.d.ts",
    "sideEffects": false
}

```

Once I got by those errors, I had an error "no export named default" importing Tailwind using this command
import Tailwind from "primereact/passthrough/tailwind";

I changed index.esm.mjs  in node_modules/primereact/passthrough/tailwind from this

```
module.exports = {
  Tailwind: Tailwind
};

export { TRANSITIONS };

```
to this

`export { TRANSITIONS, Tailwind };`

and imported it as a named import.  I then had to delete the .vite/deps directory and recompile and it seems to be working.

I probably created other problems that I havent run into yet, but just wanted you to know.   If there is another way to get around these issues, please let me know.   

Thanks,
Bill

**Comment by melloware** [comment, marked as answer]
Fixed with 10.0.2
